### PR TITLE
added dblp directory and tutorial section

### PR DESCRIPTION
**Pull request by mspector**

### CODE REVIEW

File: dblp/python/superuser-init.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                'value-regex': '[\\S\\s]{0,5000}',
+                'required':False
+            },
+            'authors': {
+                'description': 'Comma separated list of author names, as they appear in the paper.',
+                'order': 3,
+                'value-regex': '[^,\\n]*(,[^,\\n]+)*',
+                'required':False
+            },
+            'author_emails': {
```
Comment (melisabok) [inline dblp/python/superuser-init.py]: We should change this field to authorids

File: dblp/python/superuser-init.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+
+    reply = {
+        'forum': None,
+        'replyto': None,
+        'readers': {
+            'description': 'The users who will be allowed to read the above content.',
+            'values': ['everyone']
+        },
+        'signatures': {
```
Comment (melisabok) [inline dblp/python/superuser-init.py]: signatures and writes will be DBLP.org/upload, I think we should create this group too,

**Comment by andrewmccallum** [comment]
I think there should be both "authors" and "authorids"
The first are their string names, exactly as they appear in the DBLP BibTeX record.
The second will be email address, with "null" for authors for whom we don't have an address.

**Comment by mspector** [comment]
yes, this authors and authorids are both included

**Pull request merged**